### PR TITLE
Add description to roadmap items

### PR DESCRIPTION
**Pull request by aromeronavia**
## Things done in this PR

1. Made a small refactor in the `Github Issue` entity to differentiate between the description and the issue body since the organization name and roadmap name was pulled from the `description` field from the `Github Issue` entity, but it was not aligned with the real description is, so made a rename in there. Let me know what you think.

## Important stuff with type ids

Seems like each roadmap has a different `Type ID` for the items, so I believe we should make it configurable via an Env variable (?) 

I left the type id of my roadmap in the diff so you can see it, but will remove.

### CODE REVIEW

File: README.md
Comment (iaserrat) [inline README.md:53]: This roadmap template is something users can create in Wizeline?

**Review by iaserrat — APPROVED** [review]
LGTM

We need to find a way to automate the retrieval of the Unit ID 🤔

**Comment by iaserrat** [comment]
Ping @Insane9

**Comment by iaserrat** [comment]
Can we merge?

**Comment by aromeronavia** [comment]
:shipit:

**Comment by aromeronavia** [comment]
Just waiting for another review before merge :+1:

**Review by cristina-9 — APPROVED** [review]
all good

**Comment by aromeronavia** [comment]
Will solve the conflicts and merge :+1:

**Pull request merged**
